packages/core/admin/ee/server/routes/review-workflows.js

```diff
@@ -119,7 +119,7 @@ module.exports = {
           {
             name: 'admin::hasPermissions',
             config: {
-              actions: ['admin::users.read', 'admin::review-workflows.read'],
+              actions: ['admin::users.read'],
```

Comment:
imo this should not be based on review workflows read permissions, wdyt @jhoward1994 @nathan-pichon

Reply:
Yes that makes sense to me. That permission is more about interactions with the RW settings page

Reply:
indeed, maybe we could work on that when we discuss RBAC

Reply:
could you elaborate a bit more?

Reply:
It has been named "read" to be coherent with others permissions, but I still think it would have make more sense if it was named "access" or else.

Reply:
oh, alright :) I will keep it then. Do you think this PR is necessary then? maybe we can only keep that `admin::review-workflows.read` permission for now.

Reply:
Until we have proper RBAC permissions, it might not be useful.
But I think we will need
```javascript
if (permissionChecker.cannot.update()) {
    return ctx.forbidden('You cannot update this entity assignee');
}
```
at some point when, we will be able to assign roles to atomic actions like update assignee or a stage on entities..

Reply:
👍  for now this would only prevent updating assignee if you are not allowed to update any entity of the content type. Alright, I will close this for now ! 🔐